packages/edit-site/src/components/post-fields/index.js

```diff
@@ -179,26 +150,22 @@ function usePostFields( viewType ) {
 	const { records: authors, isResolving: isLoadingAuthors } =
 		useEntityRecords( 'root', 'user', { per_page: -1 } );
 
-	const { frontPageId, postsPageId } = useSelect( ( select ) => {
-		const { getEntityRecord } = select( coreStore );
-		const siteSettings = getEntityRecord( 'root', 'site' );
-		return {
-			frontPageId: siteSettings?.page_on_front,
-			postsPageId: siteSettings?.page_for_posts,
-		};
-	}, [] );
+	const { frontPageId, postsPageId, getFeaturedMediaUrl } = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( coreStore );
+			const siteSettings = getEntityRecord( 'root', 'site' );
+			return {
+				frontPageId: siteSettings?.page_on_front,
+				postsPageId: siteSettings?.page_for_posts,
+				getFeaturedMediaUrl: ( id ) =>
+					getEntityRecord( 'root', 'media', id ),
+			};
+		},
+		[]
+	);
 
 	const fields = useMemo(
 		() => [
-			{
-				id: 'featured-image',
-				label: __( 'Featured Image' ),
-				getValue: ( { item } ) => item.featured_media,
-				render: ( { item } ) => (
-					<FeaturedImage item={ item } viewType={ viewType } />
```

Comment:
Two questions:
1. Was there a reason to change the order here?
2. What was the reason for inlining the `render` function instead of keeping the `FeaturedImage` component?